lib/DDG/Spice/BrainyQuote.pm

```diff
@@ -26,6 +26,11 @@ handle remainder => sub {
     if(/^\w\.\w/) {
         s/\./\. /g;
     }
+    
+    # Avoid triggering on 'stock' quotes; these are handled by Stocks IA
+    if ($req->query_lc =~ m/stock quote/) {
+       return;
+    }
```

Comment:
Hmm, yes.

@moollaza @iambibhas , is there a significant gain in using that method, or does ```$req->query_lc``` do the trick? Disclaimer: new to Perl. :P